src/iad.cs

```diff
@@ -172,7 +172,8 @@ interface ADInterstitialAdDelegate {
 		void WillLoad (ADInterstitialAd interstitialAd);
 	}
 
-	[Category, BaseType (typeof (MPMoviePlayerController))]
+	[Category (ignoreStaticWarnings: true)]
```

Comment:
We should somehow make sure these are fixed in XAMCORE_4_0.

One easy way to do this would be to make the attribute field an error:

    #if XAMCORE_4_0
    [Obsolete ("Inline the static members in this category in the category's class (and remove this obsolete once fixed)"]
    #endif
    public CategoryAttribute (bool ignoreStaticWarnings)
    {
        // ....
    }